docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+---
+title: "Make an LED Blink With Buttons And With Code"
+linkTitle: "Blink an LED"
+weight: 30
+type: "docs"
+description: "Use the Viam app and the SDKs to make an LED turn on and off with a Raspberry Pi."
+webmSrc: "/img/blink.webm"
+mp4Src: "/img/blink.mp4"
+videoAlt: "A blinking L.E.D. connected to a Raspberry Pi"
+images: ["/img/blink.gif"]
+aliases:
+    - /tutorials/make-an-led-blink-with-the-viam-app/
+    - /tutorials/make-an-led-blink-with-a-raspberry-pi-and-sdk/
+    - /tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk/
+    - /tutorials/get-started/make-an-led-blink-with-the-viam-app/
+tags: ["board", "raspberry pi", "app", "sdk"]
+authors: []
+languages: [ "python", "go" ]
+viamresources: [ "board" ]
+level: "Beginner"
+date: "4 October 2022"
+cost: 70
+no_list: true
+---
+
+This tutorial will show you how to use Viam to make an LED blink with a Raspberry Pi.
+This is a great place to start if you have never built a robot or a circuit before.
+
+{{<gif webm_src="/tutorials/img/blink-an-led/image9.webm" mp4_src="/tutorials/img/blink-an-led/image9.mp4" alt="A GIF of the completed project showing a blinking blue LED connected to a Raspberry Pi with jumper cables." max-width="300px">}}
```

Comment:
I think the video from the other tutorial is actually nicer because it's not jumpy?

Reply:
I kinda like the lighting etc. on this one so I left it in, yeah. And then the other ones are near the part of the tute that talks about coding them so that it's actually a set interval. Can change if you want though?